### ISSUE TITLE
[MIGRATED] Develop modules for Feature Phone users in Nigeria and South Africa

### ISSUE THREAD

**Issue by dajbelshaw** [mozilla/teach.webmaker.org#90]
_From @LauraHilliger on January 18, 2015 8:33_

We are partnering with Every1Mobile to #TeachTheWeb to feature phone users in Nigeria and South Africa. This is a pilot project.

**Target Audience 18 - 24 year olds** in these countries. Potentially millions of people. Due to connection in these countries, and the fact that we are developing for a feature phone, our modules for the pilot will be text based with limited interactivity. Typical functionality for these modules could be:

* articles, modules, longer textbook type text
* quizzes
* opinion and experience tools ("do you like this?" If [user] says yes, then go to page X)
* creative expression (text based)
* virtual q and a's, comment threads, forums
* P2P -> young people can post questions and receive answers, very participatory
* social features

We will need to create a (mini) module for

* Navigation
* Basic Search
* Data Storage

So the question is, how can you improve someone's web literacy on these three topics using a text-based curriculum displayed on feature phones? Let's get creative! 

cc/ @keyboardkat @dajbelshaw @iamjessklein @benrito

[Project notes are here.](https://foundation.etherpad.mozilla.org/every1mobile)

_Copied from original issue: mozilla/webmaker-curriculum#108_

**Comment by dajbelshaw** [comment]
Cool. I seem to remember from my mobile learning days that Geoff Stead (now at Qualcomm) did some stuff around mxit in South Africa. I'll ask him and John Traxler for info.

**Comment by dajbelshaw** [comment]
_From @geoffstead on January 20, 2015 2:46_

Hi there - thanks for looping me in, Doug.

Several related tools / projects / teams I would review in this are:
1) mxit is a huge social media platform in southern africa, and works on old-school phones as well as smart ones. Many high profile learning and outreach initiatives have been run on it (and no doubt still are). every1Mobile might well be using them. (if not, make sure you are not missing a key audience). 

2) another very interesting platform is BiNu http://www.binu.com/ - like mixit they offer smart phone like features to non-smart phones, and are hugely successful in the developing world. See especially their collaboration with WorldReader (http://www.worldreader.org) who use them to distribute ebooks to feature-phones

and then some organisations using these and similar platforms to deliver real learning outreach

3) m-ubuntu (http://www.m-ubuntu.org.za/) is an amazing, yet small, team who do very grass routes teacher training / classroom support across rural south africa, using whichever mobile tech they can lay their hands on. They have the most solid, and road-tested pedagogies I've seen in "development projects"

4) rlabs is a social-good learning team, who often partner with mxit and offer a range of training events: http://www.rlabs.org/tag/mxit/. Very highly regarded

5) several of South Africa's academic institutions are active in this area too:
- meraka institute
- unisa (tom brown)
- north west university (herman van der merwe)

I also run an (occasional) blog, tracking related projects: http://moblearn.blogspot.com/search/label/africa

hope this helps

**Comment by dajbelshaw** [comment]
This is *great* stuff, @geoffstead  - thanks so much! Knew you'd be super-helpful. :-D

**Comment by dajbelshaw** [comment]
_From @Algywilliams on January 22, 2015 11:17_

Thanks @Geoffstead. It's Algy from Every1Mobile here. Yes, we work very closely with Mxit and biNu - other platforms that we use for distribution are 2Go, Eskimo and Opera.

**Comment by dajbelshaw** [comment]
Just want to query the 30 January milestone given amount of stuff mentioned by @LauraHilliger in first comment?

**Comment by dajbelshaw** [comment]
_From @benrito on January 24, 2015 14:10_

Yes, we'll need to discuss. Doug, I'll find a time early this week.

**Comment by dajbelshaw** [comment]
_From @thornet on January 27, 2015 11:10_

@benrito & @Algywilliams are confirming timelines. This will likely be kicked off in earnest in a few weeks, so I'm changing the milestone date.

**Comment by dajbelshaw** [comment]
Met with @benrito and Katherine van Wyk (E1M) today. Katherine is going to send details of an information architecture that takes into account the constraints of their platform / feature phones. Basing this off the [Kraken activity](https://mozilla.github.io/webmaker-curriculum/WebLiteracyBasics-I/session01-kraken.html) in the first instance. Should have this by the end of the week. :)

**Comment by dajbelshaw** [comment]
Update: follow-up meeting postponed until Wednesday 11th Feb as E1M need extra time to get Information Architecture to us.

**Comment by dajbelshaw** [comment]
Update: met with Katherine, Alex & Algy from Every1Mobile. Went through information architecture for pilot activities on featurephones.
![wireframe_digital literacy course draft_v0 1](https://cloud.githubusercontent.com/assets/2429426/6147881/d46c85ee-b1f3-11e4-8d0c-882930e27d8b.png)

**Comment by LauraHilliger** [comment]
@benrito I haven't heard about this since I've been back. Can I assume this is safe to close?

**Comment by benrito** [comment]
Hey partner—we are waiting on some work from E1M. It’s not prioritized in any current Heartbeat but please leave open for time being. Thank you!

**Comment by toolness** [comment]
<a href="https://github.com/mozilla/learning-networks/issues/41"><img src="https://raw.githubusercontent.com/toolness/learning-issue-migrator/master/img/migrated.png"></a>
This issue has been moved to https://github.com/mozilla/learning-networks/issues/41.
Please don't comment on it here.
